src/index.html

```diff
@@ -1,13 +1,468 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>Crazybaby</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="icons/favicon2.svg"
+    >
+    <link rel="stylesheet" href="styles/styles.scss">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+  <body class="page__body">
+    <header class="header" id="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar-logo">
+              <img
+                src="./icons/logo.svg"
+                alt="Crazylab logo"
+                class="header__logo"
+              >
+            </a>
+
+            <div class="top-bar__icons top-bar-tablet">
+              <div class="top-bar__additional-tablet">
+                <a class="top-bar__specs">SPECS</a>
+                <p class="top-bar__language">
+                  <a class="top-bar__language-en">EN</a> / <a class="top-bar__language-ua">UA</a>
+                </p>
+              </div>
+
+              <a class="top-bar__btn-tablet">Buy</a>
+
+              <a href="#menu" class="icon icon--menu"></a>
```

Comment:
for links, we use the tag `<a/>`, for buttons we use the tag `<button/>`,
so if the redirect does not occur, then you should use the `button`